e2e/cypress/integration/channel/system_message_spec.js

```diff
@@ -56,14 +56,16 @@ describe('System Message', () => {
     });
 
     it('MM-14636 - Validate that system message is wrapping properly', () => {
+        const newHeader = `${Date.now()} newheader`;
+
         // # Update channel header textbox
-        cy.updateChannelHeader('> newheader');
+        cy.updateChannelHeader(`> ${newHeader}`);
 
         // * Check the status update
         cy.getLastPost().
             should('contain', 'System').
             and('contain', '@user-1 updated the channel header from:').
```

Comment:
Hi @saturninoabril 

This test expects an existing header to be present on a channel. Only then, when we update the header, we would expect the text `@user-1 updated the channel header from:`. If the Channel header is changed for the first time (this can happen when the test ran first time), then the expected message would be `@user-1 updated the channel header to:`. 

When I ran the test on Cypress instance, it failed because of the same reasons:
![image](https://user-images.githubusercontent.com/1429138/62546735-aa5ca100-b881-11e9-87ba-5fe633f10997.png)